lib/kernel/src/application_controller.erl

```diff
@@ -528,19 +528,18 @@ check_conf_data([]) ->
 check_conf_data(ConfData) when is_list(ConfData) ->
     [Application | ConfDataRem] = ConfData,
     case Application of
-	{kernel, List} when is_list(List) ->
-	    case check_para_kernel(List) of
-		ok ->
-		    check_conf_data(ConfDataRem);
-		Error1 ->
-		    Error1
-	    end;
 	{AppName, List} when is_atom(AppName), is_list(List) ->
-	    case check_para(List, atom_to_list(AppName)) of
-		ok ->
-		    check_conf_data(ConfDataRem);
-		Error2 ->
-		    Error2
+	    case lists:keymember(AppName, 1, ConfDataRem) of
+		true ->
+		    erlang:display({warning, "duplicate application config: " ++
```

Comment:
I would really like this to use ?LOG_WARNING instead of erlang:display. Did you consider that? I mean, it's early in the startup, but the logger process is already running with the simple handler. The simple handler does erlang:display, but next step is that it is replicated to the default handler (if such is configured), which will make it more visible. Any thoughts?

Reply:
Oh, I did not know that (about using erlang:display during boot). That's much better indeed!

Reply:
Yeah, the simple handler (logger_simple_h.erl) is added when the logger process starts, and buffers up to 10 messages which it will replay on the default handler if such is started later on. So, if you don't mind, just go ahead with that.

Reply:
@sirihansen I have pushed a new commit for your review. You can look at the commit message for the new output. Meanwhile, I will work on set_env/1 and merge those changes in there.